doc/source/templates/04_finetuning_llms_with_deepspeed/compute_configs/aws_7b_or_13b.yaml

```diff
@@ -1,22 +1,16 @@
-# 1 g5.16xlarge + 15 g5.4xlarge --> 16 GPUs, 256G RAM on trainer and 64G RAM on workers
+# Autoscale to 16 g5.4xlarge --> 16 A10Gs
 region: us-west1
 allowed_azs: [any]
 head_node_type:
     name: head_node
-    instance_type: g5.16xlarge
-    resources:
-      custom_resources: 
-        large_cpu_mem: 1
+    instance_type: m5.xlarge
```

Comment:
Can we bump the size of the headnode. 4 CPUs is a bit small. the driver still does some nontrivial stuff (writing to a bunch of files, etc.)